docs/FORGER.md

```diff
@@ -0,0 +1,115 @@
+# Forger Node
+
+## Setup
+
+--- 
+
+Run the init.sh script to initialize the deployment for the first time. Select **forger** node and the **network** to run (eon or gobi).
+
+```shell
+./scripts/init.sh
+```
+
+The script will generate the required deployment files under the [deployments](../deployments) directory and provide instructions on how to run the compose stack.
+
+## Zend seed
+
+--- 
+
+As this is a forger node, it requires a zend node to run as well. Syncing a zend node from scratch may take a few hours, 
+so a seed file can be used to speed up the process.
+
+There are two variables that help controlling the seed process for the zend node:
+
+- `ZEN_USE_SEED_FILE`: The default value for this variable is `false`, but it can be set to true choosing to use the seed file in the init.sh script.
+
+- `ZEN_FORCE_RESEED`: This variable is set to `false` by default, but it can be set manually to `true` if forcing the reseed of the zend node is required.
+
+Once the zend node has been synced, the following times the stack is started there is no need to seed the zend node again.
+
+The _seed.sh_ script will generate a **.seed.complete** file in the node's data folder, and it will prevent the seed script to be run again.
+
+Additionally, setting the `ZEN_USE_SEED_FILE` variable to false in the .env file will prevent the seed process to be run again.
+
+If migrating from previous versions of these project and your zend node is already synced set the `ZEN_USE_SEED_FILE` variable to false in the .env file.
+
+If reseed of the zend node is required, set the `ZEN_FORCE_RESEED` variable to true in the .env file, and restart the stack.
+
+## Running the stack
+
+--- 
+
+1. Prerequisites
+    - Storage: A minimum of 250GB of free space is required to run both nodes. Keep in mind that the storage requirements will grow over time.
```

Comment:
It could be useful to mention, instead of "both nodes", that we're referring to run "a forger node for both, testnet and mainnet network", just to not get people confused on running "both rpc and forger node".